doc/api/v8.md

```diff
@@ -1304,6 +1304,26 @@ setTimeout(() => {
 }, 1000);
 ```
 
+## `v8.isOneByteRepresentation(content)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `content` {string}
+* Returns: {boolean}
+
+Returns `true` if `content` uses one byte as the underlying representation Otherwise, returns `false`.
```

Comment:
This should explain that:
- The underlying representation is ISO-8859-1 specifically
- That the underlying representation isn't just a single byte (which is what this currently says, but which obviously wouldn't be very useful)
- That this method can return `false` even for strings that are entirely representable in ISO-8859-1